### ISSUE TITLE
Restoring a given revision of a folder or file and choosing the destination in which to restore

### ISSUE THREAD

**Issue by sunknudsen** [arqbackup/arq_restore#44]
Hi,

Huge fan of Arq.

Trying to put together a standalone backup strategy that doesn’t rely on having a fat client installed to restore a folder or file (say I need to restore on someone’s computer).

Love the concept of adding the `arq_restore` binary to the backup media (makes it standalone).

Is it possible to restore a given revision of a folder or file instead of the latest? I ran into an edge case recently where a macOS update deleted the keys to a provisioning profile. Latest revision no longer included the keys. Had to restore from an earlier revision.

Is it possible to choose the destination in which to restore instead of where the source file was to prevent overwriting?

Couldn’t find documentation for the above.

Thanks,

Sun

**Comment by sreitshamer** [comment]
Hi Sun,

There isn’t a feature for selecting a different backup record.
Use the Arq app for that.
You can package the Arq app in your backup media. It doesn’t require a license code or anything from us in order to restore files.

- Stefan

**Comment by sunknudsen** [comment]
Hi Stefan,

Thanks for the follow up.

> You can package the Arq app in your backup media. It doesn’t require a license code or anything from us in order to restore files.

Wouldn't this approach save the encryption key in Keychain and perhaps also some preference files on the host computer?

I like how concealed the `arq_restore` approach appears to be.

Would you consider adding these features?

Best,

Sun

**Comment by sreitshamer** [comment]
No, the app bundle doesn’t include your keychain entries or preferences files.
